CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
+## Unreleased as of Sprint 71 ending 2017-10-16
+
+### Added
+- Expose disks and volumes in the ServiceModel through Hardware. [(#91)](https://github.com/ManageIQ/manageiq-automation_engine/pull/91)
+- Ansible Playbook Method. [(#89)](https://github.com/ManageIQ/manageiq-automation_engine/pull/89)
```

Comment:
@JPrause Let's drop this one `Ansible Playbook Method. (#89)` as it is really a cleanup PR and is confusing listed here along with `Ansible Playbook Automate Method` which was the bulk of the work.